db/migrate/20210715161256_change_add_doi_default.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeAddDoiDefault < ActiveRecord::Migration[6.1]
+  def change
+    change_column_default :works, :assign_doi, true
```

Comment:
Is this required? I'm concerned that this will affect all existing works.  We don't want this value to be true for works that were create without DOIs

Reply:
Yes, it is required. The default is supposed to be true when depositor selects. Note that this is combined with the collection DOI option to determine whether a DOI is assigned. (That is, it is ignored when DOI option is yes or no.)

Reply:
@justinlittman Can we move the default off of the model and onto the form?

Reply:
Hmm, probably not since its a boolean.  Can/should we set the existing works to "DOI requested" or should they be "DOI refused"?

Reply:
Existing works would be "no", which makes sense since the user did not request one or have the option of choosing. However, for future works the default would be "yes", but the user would have the option of refusing.

Reply:
@justinlittman what I'm saying here is that by changing this default, it changes all of the existing to "yes"

Reply:
Oh, actually, the other migration runs first, which sets them to "false", and only then is the default chagned, so this should be fine.